src/App.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Routes, Route, useLocation, Navigate,
+} from 'react-router-dom';
+import './Style/register.css';
+import './Style/experienceDetails.css';
+import './Style/medias.css';
+import { loadUserFromLocalStorage } from './Redux/Slices/usersSlice';
+import LoginPage from './Components/LoginPage';
+import Register from './Components/Register';
+import RoutesWrapper from './Components/RoutesWrapper';
+import Navbar from './Components/NavBar';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your main app component will meet the latest standards. 💪

- [ ] OPTIONAL: Refactor the rest of the functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 13):

![image](https://github.com/HFG43/final_capstone_front_end/assets/57421823/1aac58a4-654a-4325-b886-3a54baf818fe)

That way, all your code will meet the latest standards. 💪